data/strings/strings.txt

```diff
@@ -16483,7 +16437,7 @@
     zh-Hant = 下載地圖來尋找位置和離線瀏覽。
 
   [stop]
-    tags = android,ios
+    tags = ios
```

Comment:
@kirylkaveryn shoudn't iOS use `navigation_stop_button` translation instead of this one in NavigationViewController.xib?
`<userDefinedRuntimeAttribute type="string" keyPath="localizedText" value="stop"/>`

Reply:
I'll fix it